test/SILOptimizer/address_lowering_misc.sil

```diff
@@ -0,0 +1,31 @@
+// RUN: %target-sil-opt -address-lowering -enable-sil-opaque-values -sil-verify-all %s | %FileCheck %s
```

Comment:
I used the `_lib` suffix for tests that need to import the stdlib. as in `test/SILGen/opaque_values_silgen_lib.swift`